### ISSUE TITLE
Bibliography API Issue

### ISSUE THREAD

**Issue by toolCHAINZ** [manjavacas/typslides#2]
Hello!

Thank you for this package, it's lovely!

I've run into a small issue with the bibliography slide. When I try to use it like so:

```typst
#bibliography-slide("refs.bib")
```

I get an error:

```
error: file not found (searched at /Users/<username>/Library/Caches/typst/packages/preview/typslides/1.2.1/refs.bib)
    ┌─ @preview/typslides:1.2.1/typslides.typ:349:6
    │
349 │       bib-path,
    │       ^^^^^^^^

```

Reading through the relevant [typst documentation](https://typst.app/docs/reference/syntax/#paths-and-packages), it seems this is an intentional part of typst's package design:

> A package can only load files from its own directory. Within it, absolute paths point to the package root, rather than the project root. For this reason, it cannot directly load files from the project directory. If a package needs resources from the project (such as a logo image), you must pass the already loaded image, e.g. as a named parameter logo: image("mylogo.svg"). Note that you can then still customize the image's appearance with a set rule within the package.

You might want to rethink this particular API as I'm not sure if it's realistically usable as it is.

**Comment by manjavacas** [comment]
Hi @toolCHAINZ!

While testing this type of slide, I noticed this issue, but I decided to leave it as-is in case someone wants to use Typslides locally.  

That said, it might cause some confusion and affect the experience, so I’ll make sure to add a note about it in the README.  

Hopefully, future versions of Typst will improve compatibility between package templates and external files. Fingers crossed—we’ll keep waiting!  

Thanks for pointing it out! Glad to hear you’re enjoying Typslides. 😉

**Comment by manjavacas** [comment]
There seems to be a way to use `#bibliography-slide` via API by making use of a call to `bibliography(...)` instead of the `bib-path` (see https://github.com/typst/packages/pull/1381#issuecomment-2516876444).

I have included this functionality in version 1.2.3.
